backend-services/user-verification-service/src/main/java/org/cardano/foundation/voting/domain/entity/SMSUserVerification.java

```diff
@@ -23,8 +24,12 @@ public class SMSUserVerification extends AbstractTimestampEntity {
     @Column(name = "id", nullable = false)
     private String id;
 
-    @Column(name = "stake_address", nullable = false)
-    private String stakeAddress;
+    @Column(name = "wallet_id", nullable = false)
+    private String walletId;
```

Comment:
`stakeAddress` is now `walletId`. Also `walletidType` was added to specify the wallet type (CARDANO or KERI)